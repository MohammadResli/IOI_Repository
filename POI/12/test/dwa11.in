200
7 88 114 107 109 146 169 33
11 133 175 190 5 198 197 55 125 41 74 12
12 27 17 79 58 67 131 64 66 52 88 33 195
7 196 50 136 46 148 62 143
6 94 28 56 2 85 29
7 72 14 166 80 156 169 138
7 12 125 109 102 169 90 113
12 152 72 175 37 93 10 80 148 15 162 90 70
7 62 15 171 39 140 81 20
14 58 64 114 88 104 138 20 8 85 171 179 60 57 43
8 119 166 99 134 124 146 44 55
14 67 34 84 96 132 139 119 17 196 7 110 16 175 2
8 25 129 20 40 21 97 34 31
11 55 138 145 190 6 162 110 63 94 29 15
19 127 176 114 103 196 108 168 49 8 80 41 35 9 111 105 166 14 132 91
12 148 68 47 115 178 197 38 120 109 177 56 12
9 12 3 87 49 188 175 122 140 169
6 193 66 127 147 70 81
14 38 120 94 49 105 62 42 176 149 146 34 170 125 30
8 143 9 10 141 131 13 72 81
14 165 34 111 113 166 51 101 13 41 152 22 108 80 32
13 156 121 196 132 136 106 153 25 166 164 21 145 24
14 93 80 178 55 66 50 30 102 173 106 170 155 87 144
8 61 39 22 132 62 126 97 94
8 182 28 13 102 52 22 189 151
10 57 108 92 149 139 148 138 54 96 67
11 184 172 145 61 193 34 119 64 3 146 199
8 119 25 5 60 125 106 70 189
9 35 56 60 188 80 169 131 14 5
12 191 53 92 71 33 46 192 19 156 73 69 23
9 190 139 101 60 89 13 93 43 183
6 130 181 21 113 41 95
11 1 71 85 104 191 126 3 183 91 30 98
10 21 27 19 172 61 150 12 13 119 41
15 54 177 146 77 120 29 106 122 121 15 76 154 144 194 158
13 44 196 174 132 182 41 123 89 60 49 73 193 162
13 199 89 144 173 8 79 113 174 43 103 97 116 190
17 88 153 90 156 179 44 49 195 141 16 42 41 183 39 77 53 19
11 100 92 160 163 24 102 115 174 9 54 38
10 110 75 117 192 114 53 13 151 180 129
18 15 21 34 117 124 168 175 108 36 158 170 32 113 187 2 91 120 38
8 46 19 167 104 179 199 38 192
13 106 130 31 150 58 177 71 158 111 37 97 10 107
14 140 80 100 181 138 177 38 169 114 162 36 190 173 11
6 51 185 145 109 76 154
8 42 97 60 101 144 4 30 143
14 96 60 197 78 173 133 151 172 16 168 107 152 59 113
3 66 199 193
12 187 19 84 15 65 97 38 36 17 89 197 109
9 55 122 95 188 194 114 117 23 4
6 191 21 168 45 71 172
9 103 3 93 163 172 25 165 131 132
11 113 38 179 140 138 177 150 40 30 148 171
8 119 35 92 161 39 173 192 26
12 101 140 104 73 87 11 181 2 50 23 192 14
12 130 163 5 119 67 197 98 124 135 166 16 29
10 158 10 157 163 26 124 71 60 177 147
10 178 140 102 69 10 160 3 156 43 179
12 141 104 65 197 164 47 72 180 75 70 82 84
17 28 31 187 10 57 129 154 36 92 117 116 29 190 198 146 47 46
9 27 116 159 24 100 180 34 127 126
13 186 19 9 142 24 108 146 140 144 64 184 166 4
6 80 14 188 72 114 186
9 133 99 10 27 114 85 152 62 3
10 142 195 154 59 156 78 138 177 170 49
9 18 121 139 48 192 127 76 23 3
12 141 117 12 75 96 70 179 56 110 80 26 3
12 130 180 181 138 90 113 71 16 157 79 139 85
8 83 58 178 30 109 85 134 200
14 28 74 118 72 67 160 81 59 186 180 18 197 8 139
9 137 51 33 43 173 57 68 30 127
12 80 133 20 93 155 8 63 59 150 6 70 191
10 55 105 96 112 36 141 103 91 30 108
9 126 91 175 70 105 89 2 116 88
9 121 131 159 40 175 162 59 67 117
6 146 117 66 138 45 35
16 165 35 38 81 84 182 141 187 129 170 117 124 131 102 115 163
10 47 108 153 167 94 171 142 133 163 65
8 3 190 164 168 68 90 126 37
11 23 117 21 63 8 15 72 44 6 29 67
13 198 18 70 132 100 91 133 140 77 185 9 199 20
11 156 169 110 83 59 126 178 136 164 174 189
12 140 123 153 130 82 154 141 104 118 117 69 94
5 59 77 49 99 12
12 33 190 102 10 98 199 64 112 177 5 69 68
6 161 107 171 100 88 155
8 117 55 191 17 23 139 143 193
12 74 10 38 86 194 168 3 200 182 1 94 196
9 169 37 134 31 36 49 176 170 74
11 129 166 187 195 68 8 154 38 79 179 7
10 81 41 33 122 163 74 153 15 151 73
7 159 54 39 135 26 60 30
9 72 125 170 144 31 106 8 52 23
13 78 24 199 97 108 166 135 83 5 19 141 14 88
8 135 116 146 50 155 120 142 32
8 47 125 26 67 174 12 188 73
14 109 49 119 184 194 157 24 191 37 46 94 13 168 43
13 56 163 130 168 182 128 195 85 33 199 133 196 123
6 11 64 136 84 159 195
11 122 127 137 39 111 44 81 149 61 184 86
8 21 118 31 110 46 105 199 55
14 170 25 119 160 7 23 143 190 39 140 77 85 58 151
8 52 132 119 156 135 15 73 37
11 106 55 59 162 186 33 10 42 122 83 179
9 134 74 169 187 19 101 73 15 189
11 122 120 22 23 28 35 147 104 43 93 161
12 119 117 177 86 146 47 118 1 149 43 154 188
9 94 62 166 15 26 73 41 78 21
11 49 186 16 124 69 1 161 162 97 45 7
8 101 151 82 40 12 111 67 14
9 135 21 115 15 100 43 127 118 110
5 73 85 192 190 153
12 125 21 47 32 166 124 68 37 7 41 159 53
14 15 122 1 126 10 188 179 64 40 198 63 44 141 50
7 154 39 16 135 143 77 111
9 181 37 61 74 163 138 95 135 60
17 157 130 60 75 76 40 50 67 77 107 179 190 80 87 83 41 142
14 83 129 135 185 176 101 107 133 187 120 70 111 171 175
15 143 11 158 12 102 151 27 107 34 54 56 103 199 97 28
10 16 35 106 169 95 19 126 170 41 118
5 22 75 66 141 35
19 17 35 180 100 187 106 132 91 161 139 198 104 50 114 165 126 178 154 125
6 154 129 83 98 36 157
9 113 11 109 169 184 57 77 41 56
13 2 133 168 171 93 96 28 141 113 135 19 7 122
11 82 197 24 33 122 61 74 120 114 174 79
14 15 71 199 66 134 100 18 152 61 188 111 133 139 145
4 151 142 98 195
9 13 40 188 60 77 171 123 118 90
10 43 83 144 56 155 32 68 117 169 98
15 75 183 159 77 185 194 20 3 166 52 139 143 197 160 29
13 153 52 155 81 22 122 173 36 103 24 12 146 15
12 125 98 118 64 177 176 72 47 2 127 81 78
11 11 190 193 69 105 153 89 174 127 169 162
14 125 118 95 115 141 92 163 178 56 94 111 175 116 103
7 155 177 156 4 82 99 22
5 168 195 71 100 171
11 53 6 26 65 44 195 14 116 68 76 10
11 68 87 151 70 12 131 66 26 122 127 31
11 83 81 58 44 62 157 9 17 55 53 102
14 83 67 114 20 59 174 125 38 121 94 135 77 147 73
8 65 173 117 78 170 128 95 62
14 146 87 46 197 145 191 20 102 158 115 4 131 196 119
11 35 62 23 163 176 46 152 130 93 37 181
8 179 45 27 14 143 157 127 22
13 62 1 60 19 132 27 76 11 35 95 200 107 143
9 18 169 106 165 192 156 57 197 141
7 16 199 26 53 4 186 8
7 107 158 173 100 26 165 19
4 34 53 72 43
13 91 196 25 176 128 191 119 110 47 102 139 187 40
12 64 193 8 21 47 162 175 127 161 178 144 192
12 112 156 157 38 132 134 189 159 22 78 83 91
14 65 175 90 123 35 45 115 156 83 160 177 107 60 122
9 130 136 23 95 72 185 86 132 184
14 184 30 65 38 157 136 58 6 22 147 82 103 154 153
9 156 57 68 153 123 140 145 117 97
9 43 57 119 41 149 166 195 143 35
10 99 153 181 131 75 174 92 163 113 61
7 102 168 154 58 131 39 70
9 152 109 122 54 182 86 179 106 188
10 134 14 8 75 44 193 152 36 109 104
13 77 116 159 91 56 78 135 39 98 57 144 190 52
6 178 22 79 82 59 174
10 77 149 122 191 147 187 170 52 167 21
13 94 108 158 131 15 90 113 11 6 21 56 62 22
6 78 165 184 42 179 182
13 137 160 194 88 51 97 125 47 15 41 79 98 197
16 7 147 124 1 134 44 120 186 82 130 6 17 105 173 29 89
12 142 120 89 23 77 165 65 41 93 19 200 102
9 118 125 137 10 9 78 86 129 53
7 185 34 47 52 27 51 173
15 23 172 71 142 183 190 193 149 37 47 54 44 175 132 169
11 199 159 126 37 134 141 36 39 96 164 82
13 191 152 2 17 154 8 118 12 41 135 75 173 74
9 133 118 151 144 19 15 182 193 89
12 107 35 65 57 136 16 44 85 154 43 133 53
10 186 69 16 122 82 135 164 23 152 58
13 145 161 104 38 117 90 58 67 10 42 167 114 53
8 182 70 68 59 40 122 61 185
10 44 55 144 159 195 189 116 32 185 68
9 36 176 98 25 180 161 77 167 88
5 173 31 131 33 38
8 124 100 155 62 27 97 167 156
8 118 155 81 131 172 45 181 180
8 70 148 109 63 104 62 169 178
11 105 151 41 77 90 122 118 194 60 165 49
11 17 96 114 161 129 50 29 63 107 195 127
7 105 82 28 153 181 25 192
14 117 2 14 79 37 44 134 85 60 102 173 112 163 31
10 175 151 143 51 87 33 30 72 165 97
10 189 30 55 147 66 112 42 40 152 54
10 18 134 176 152 48 87 162 36 173 27
7 88 187 97 168 131 50 35
12 65 128 98 188 38 181 99 158 90 3 138 137
9 143 36 98 22 15 4 12 88 151
12 47 143 147 49 56 70 168 126 16 2 131 59
5 122 2 81 114 60
14 81 200 174 42 148 48 98 119 94 27 101 37 127 85
5 146 170 88 199 69
