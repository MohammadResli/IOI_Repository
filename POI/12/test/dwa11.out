96
1 2 4 5 10 11 15 16 17 20 21 24 25 28 29 31 37 39 40 41 43 44 47 51 52 56 57 58 59 60 61 64 65 66 67 71 72 73 74 77 80 83 85 87 88 90 94 96 98 99 101 104 106 107 108 111 112 114 116 118 121 122 123 126 128 129 133 139 141 143 145 146 148 152 153 157 158 159 164 166 168 170 172 173 174 176 177 183 186 187 190 193 195 198 199 200 
